mlrun/runtimes/nuclio/function.py

```diff
@@ -543,11 +543,23 @@ def deploy(
         :param project:    project name
         :param tag:        function tag
         :param verbose:    set True for verbose logging
-        :param auth_info:  service AuthInfo
+        :param auth_info:  service AuthInfo (deprecated)
```

Comment:
```suggestion
        :param auth_info:  service AuthInfo (deprecated and ignored)
```
This should be noted - deprecation may mean that it's being phased out but still used. In this case it's simply ignored.